### PR TITLE
Drop SVG attribute counts

### PR DESCRIPTION
**Pull request by ddbeck**
Never count attributes again! This is a follow up to https://github.com/mdn/content/pull/5147. It changes all the remaining cases of:

> X elements are using this attribute: [long list of attribute refs]

to

> These elements use this attribute: [long list of attribute refs].

### CODE REVIEW

**Review by chrisdavidmills — APPROVED** [review]
Yup yup! Nice work.

**Comment by chrisdavidmills** [comment]
oop, little conflict to fix.

**Comment by wbamberg** [comment]
SpanishInquisitionCantCount

I hate to be picky, but to me:

 > These elements use this attribute:

... doesn't read as well as:

> The following elements use this attribute:

Something about "these/this". Is that just me?

**Comment by ddbeck** [comment]
@wbamberg 

> Something about "these/this". Is that just me?

No, not just you. I thought it was slightly awkward when I wrote it, but it was a solution (to my problem of not wanting to count more things than I have fingers) that could be achieved through an uncomplicated regular expression.

The _right_ way to do this is probably to make a section for this, like the permitted property lists we came up for the CSS recipe:

> ## Applicable elements
>
> You can use this attribute with the following SVG elements:
>
> - `elem1`
> - `elem2`
> - …
> - `elemN`

Or for cases where all elements are supported:

> ## Applicable elements
>
> You can use this attribute with any SVG element.

**Comment by wbamberg** [comment]
> The right way to do this is probably to make a section for this, like the permitted property lists we came up for the CSS recipe:

Yes, I agree (and had the same thought, after commenting of course).

**Comment by ddbeck** [comment]
I could do that with this PR, unless you think we ought to put it up for discussion more widely first.

**Comment by wbamberg** [comment]
@Rumyra , @Elchi3 , are you +1 on Daniel making the change described in https://github.com/mdn/content/pull/5148#issuecomment-845496959 ? Are there other people we should ask here?

**Comment by Elchi3** [comment]
Sounds good to me!

(I will say here that I'm not a big fan of attribute reference pages. I think if I would write SVG docs from scratch, attribute pages would sit inline or under their element page. Yes, this might mean some duplication, but at least I could then document the attribute in context of its element. I could be convinced if readers would search for attributes on their own specifically, but right now I'm not. I think readers will want to use attributes in the context of an element, so 🤷 )

**Comment by Rumyra** [comment]
Yeh looks good to me :)

**Comment by sideshowbarker** [comment]
@ddbeck This status of this PR seems to be that it’s waiting for the change described in https://github.com/mdn/content/pull/5148#issuecomment-845496959 to be made, right?

**Comment by ddbeck** [comment]
@sideshowbarker Yes, that's correct and this PR slipped my mind. I'll try to finish this today (but definitely by the end of my day tomorrow).

**Comment by ddbeck** [comment]
OK, I kinda regret that I spent as much time on this as I have today, but I've now turned these very long sentences into actual lists or a paragraph saying, "You can use this attribute with any SVG element." I tried to make this as regular as I could, across all of the SVG attribute pages. As you can see, it's a huge diff. I'm sorry.

There is something I haven't done: created a section for "Applicable elements", as previously discussed. That's kinda hard to do in a quick-and-dirty way, particularly because it's underspecified. I don't know where in the outline the section should go and I don't think the page type is sufficiently well-defined trust that I could insert the section in the right spot.

Honestly, I don't think it's a good use of my time to sort this out page-by-page, so I propose leaving it at this. Like @Elchi3 says, these pages might not be that interesting to readers anyway. If y'all think this is good enough, let's merge this (barring any immediate brokenness to deal with) and move on.

File: files/en-us/web/svg/attribute/kernelunitlength/index.html [hunk 1]
```diff
@@ -12,7 +12,12 @@
 
 <p>The <strong><code>kernelUnitLength</code></strong> attribute has two meanings based on the context it's used in. For lighting filter primitives, it indicates the intended distance for the x and y coordinates, for {{SVGElement("feConvolveMatrix")}}, it indicates the intended distance between successive columns and rows in the kernel matrix.</p>
 
-<p>Three elements are using this attribute: {{SVGElement("feConvolveMatrix")}}, {{SVGElement("feDiffuseLighting")}}, and {{SVGElement("feSpecularLighting")}}</p>
+<p>You can use this attribute with the following SVG elements:</p>
+
+<ul>
+  <li>{{SVGElement("feConvolveMatrix")}}, {{SVGElement("feDiffuseLighting")}}</li>
```
Comment (wbamberg) [inline files/en-us/web/svg/attribute/kernelunitlength/index.html:18]:
```suggestion
  <li>{{SVGElement("feConvolveMatrix")}}</li>
  <li>{{SVGElement("feDiffuseLighting")}}</li>
```

File: files/en-us/web/svg/attribute/stroke-dasharray/index.html [hunk 2]
```diff
@@ -12,9 +12,8 @@
 
 <p class="note"><strong>Note:</strong> As a presentation attribute, <code>stroke-dasharray</code> can be used as a CSS property.</p>
 
-<p>As a presentation attribute, it can be applied to any element, but it only has effect on the following twelve elements:</p>
+<p>You can use this attribute with the following SVG elements:</p>
 
-<div class="threecolumns">
```
Comment (wbamberg) [inline files/en-us/web/svg/attribute/stroke-dasharray/index.html:17]: Needs to remove the closing /div

File: files/en-us/web/svg/attribute/strikethrough-thickness/index.html [hunk 3]
```diff
@@ -38,6 +44,3 @@ <h2 id="Elements">Elements</h2>
 
 <p>The following elements can use the <code>strikethrough-thickness</code> attribute:</p>
 
-<ul>
```
Comment (wbamberg) [inline files/en-us/web/svg/attribute/strikethrough-thickness/index.html:41]: This should delete the "Elements" H2 and the para preceding this list.

File: files/en-us/web/svg/attribute/vector-effect/index.html [hunk 4]
```diff
@@ -12,7 +12,22 @@
...
+  <li>{{SVGElement("ellipse")}}</li>
+  <li>{{SVGElement("foreignObject")}}</li>
+  <li>{{SVGElement("image")}}</li>
+  <li>{{SVGElement("line")}}</li>
+  <li>{{SVGElement("path")}}</li>
+  <li>{{SVGElement("polygon")}}</li>
+  <li>{{SVGElement("polyline")}}</li>
+  <li>{{SVGElement("rect")}}</li>
+  <li>{{SVGElement("text")}}</li>
+  <li>{{SVGElement("textPath")}} {{SVGElement("tspan")}}</li>
```
Comment (wbamberg) [inline files/en-us/web/svg/attribute/vector-effect/index.html:28]:
```suggestion
  <li>{{SVGElement("textPath")}}</li>
  <li>{{SVGElement("tspan")}}</li>
```

File: files/en-us/web/svg/attribute/flood-color/index.html [hunk 5]
```diff
@@ -13,7 +13,11 @@
 
 <p class="note"><strong>Note:</strong> As a presentation attribute, <code>flood-color</code> can be used as a CSS property.</p>
 
-<p>As a presentation attribute, it can be applied to any element but it has effect only on the following two elements: {{SVGElement("feFlood")}} and {{SVGElement("feDropShadow")}}</p>
+<p>You can use this attribute with the following SVG elements:</p>
+
+<ul>
+  <li>{{SVGElement("feFlood")}} and {{SVGElement("feDropShadow")}}</li>
```
Comment (wbamberg) [inline files/en-us/web/svg/attribute/flood-color/index.html:19]:
```suggestion
  <li>{{SVGElement("feFlood")}}</li>
  <li>{{SVGElement("feDropShadow")}}</li>
```

File: files/en-us/web/svg/attribute/from/index.html [hunk 6]
```diff
@@ -10,7 +10,12 @@
 
 <p><span class="seoSummary">The <code><strong>from</strong></code> attribute indicates the initial value of the attribute that will be modified during the animation.</span> When used with the {{SVGAttr("to")}} attribute, the animation will change the modified attribute from the <code>from</code> value to the <code>to</code> value. When used with the {{SVGAttr("by")}} attribute, the animation will change the attribute relatively from the <code>from</code> value by the value specified in <code>by</code>.</p>
 
-<p>Four elements are using this attribute: {{SVGElement("animate")}}, {{SVGElement("animateColor")}}, {{SVGElement("animateMotion")}}, and {{SVGElement("animateTransform")}}</p>
+<p>You can use this attribute with the following SVG elements:</p>
+
+<ul>
+  <li>{{SVGElement("animate")}}, {{SVGElement("animateColor")}}, {{SVGElement("animateMotion")}}</li>
```
Comment (wbamberg) [inline files/en-us/web/svg/attribute/from/index.html:16]:
```suggestion
  <li>{{SVGElement("animate")}}</li>
  <li>{{SVGElement("animateColor")}}</li>
  <li>{{SVGElement("animateMotion")}}</li>
```

File: files/en-us/web/svg/attribute/in2/index.html [hunk 7]
```diff
@@ -12,7 +12,11 @@
 
 <p>The <strong><code>in2</code></strong> attribute identifies the second input for the given filter primitive. It works exactly like the {{SVGAttr("in")}} attribute.</p>
 
-<p>Three elements use this attribute: {{SVGElement("feBlend")}}, {{SVGElement("feComposite")}} and {{SVGElement("feDisplacementMap")}}.</p>
+<p>You can use this attribute with the following SVG elements:</p>
+
+<ul>
+  <li>{{SVGElement("feBlend")}}, {{SVGElement("feComposite")}} and {{SVGElement("feDisplacementMap")}}</li>
```
Comment (wbamberg) [inline files/en-us/web/svg/attribute/in2/index.html:18]:
```suggestion
  <li>{{SVGElement("feBlend")}}</li>
  <li>{{SVGElement("feComposite")}}</li>
  <li>{{SVGElement("feDisplacementMap")}}</li>
```

File: files/en-us/web/svg/attribute/xlink_colon_href/index.html [hunk 8]
```diff
@@ -14,7 +14,31 @@
...
+  <li>{{SVGElement("image")}}</li>
+  <li>{{SVGElement("linearGradient")}}</li>
+  <li>{{SVGElement("mpath")}}</li>
+  <li>{{SVGElement("pattern")}}</li>
+  <li>{{SVGElement("radialGradient")}}</li>
+  <li>{{SVGElement("script")}}</li>
+  <li>{{SVGElement("set")}}</li>
+  <li>{{SVGElement("textPath")}}</li>
+  <li>{{SVGElement("tref")}}</li>
+  <li>{{SVGElement("use")}}}</li>
```
Comment (wbamberg) [inline files/en-us/web/svg/attribute/xlink_colon_href/index.html:40]:
```suggestion
  <li>{{SVGElement("use")}}</li>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks Daniel, for your work on this. I found a few formatting typos.

While most of these pages have this bit at the start, before the example, a few have placed it later. I found:

https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/mask
https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/viewBox
https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/pointer-events
https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/transform
https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/color-profile

It's a little weird how these pages plunge right from this into the example, but when we remove all the `div id=` from the live samples we will want to add "Example" H2 headings, and that should help these pages communicate their organization better.

File: files/en-us/web/svg/attribute/strikethrough-thickness/index.html (hunk 3 above)
Comment (ddbeck) [reply files/en-us/web/svg/attribute/strikethrough-thickness/index.html:41]: Fixed with c89511d71abd9215abed41e0370e99088f4d6a57

File: files/en-us/web/svg/attribute/stroke-dasharray/index.html (hunk 2 above)
Comment (ddbeck) [reply files/en-us/web/svg/attribute/stroke-dasharray/index.html:17]: Fixed with 96599d0d99f5f237466e0e18d86f764adefd9fac

**Comment by ddbeck** [comment]
@wbamberg Thanks for the review! Fixes applied and marked resolved, where it was obvious. One outstanding question for you:
 
> While most of these pages have this bit at the start, before the example, a few have placed it later. I found:
> 
> https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/mask
> https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/viewBox
> https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/pointer-events
> https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/transform
> https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/color-profile
> 
> It's a little weird how these pages plunge right from this into the example, but when we remove all the `div id=` from the live samples we will want to add "Example" H2 headings, and that should help these pages communicate their organization better.

I wasn't quite clear on what steps you thought I should take here. Should I make all of the pages like the links above? Or change the linked pages to be like the others? Or something else?

**Comment by wbamberg** [comment]
> @wbamberg Thanks for the review! Fixes applied and marked resolved, where it was obvious. One outstanding question for you:
> 
> > While most of these pages have this bit at the start, before the example, a few have placed it later. I found:
> > https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/mask
> > https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/viewBox
> > https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/pointer-events
> > https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/transform
> > https://pr5148.content.dev.mdn.mozit.cloud/en-US/docs/Web/SVG/Attribute/color-profile
> > It's a little weird how these pages plunge right from this into the example, but when we remove all the `div id=` from the live samples we will want to add "Example" H2 headings, and that should help these pages communicate their organization better.
> 
> I wasn't quite clear on what steps you thought I should take here. Should I make all of the pages like the links above? Or change the linked pages to be like the others? Or something else?

I think that those 5 cases should follow the pattern of the other pages, and include this "You can use this attribute" bit at the start, immediately after the introductory paragraph.

Sorry to be unclear, this bit of my comment:

> > It's a little weird how these pages plunge right from this into the example, but when we remove all the `div id=` from the live samples we will want to add "Example" H2 headings, and that should help these pages communicate their organization better.

...was more of a general comment about all these pages, and not a thing I expect you to do anything about in this PR.

**Comment by ddbeck** [comment]
> I think that those 5 cases should follow the pattern of the other pages, and include this "You can use this attribute" bit at the start, immediately after the introductory paragraph.

Thank you for the clarification. Done! ✅

**Review by wbamberg — APPROVED** [review]
👍 Thanks Daniel!

**Pull request merged**